src/hooks/useDebouncedState.ts

```diff
@@ -17,7 +17,7 @@ import CONST from '@src/CONST';
  * @example
  * const [value, debouncedValue, setValue] = useDebouncedState<string>("", 300);
  */
-function useDebouncedState<T>(initialValue: T, delay: number = CONST.TIMING.SEARCH_OPTION_LIST_DEBOUNCE_TIME): [T, T, (value: T) => void] {
+function useDebouncedState<T>(initialValue: T, delay: number = CONST.TIMING.USE_DEBOUNCED_STATE_DELAY): [T, T, (value: T) => void] {
```

Comment:
@pac-guerreiro Why do we need to re-create another constant? 

If we need to do that for some reason, please also update the param name in line 9

Reply:
@DylanDylann I think you meant to tag @kacper-mikolajczak 😂

Reply:
Hi @DylanDylann, the `useDebouncedState` hook should not be tied to `SEARCH_OPTION_LIST_DEBOUNCE_TIME`. 

The `SEARCH_OPTION_LIST_DEBOUNCE_TIME` might be changed independently to `useDebouncedState` and vice versa. I thought of creating dedicated constant `USE_DEBOUNCED_STATE_DELAY` to remove unnecessary coupling.

Reply:
I like separating this even if the value is same, its better for readability and clarify